Pear/View/FeedbackView.swift

```diff
@@ -0,0 +1,118 @@
+        let path = CGMutablePath()
+        path.move(to: CGPoint(x: 0, y: size))
+        path.addLine(to: CGPoint(x:size/2, y: size/2))
+        path.addLine(to: CGPoint(x:size, y:size))
+        path.addLine(to: CGPoint(x:0, y:size))
+        let shape = CAShapeLayer()
+        shape.path = path
+        shape.fillColor = UIColor.white.cgColor
+        arrowView.layer.insertSublayer(shape, at: 0)
+        arrowbackgroundView.addSubview(arrowView)
+
+        feedbackBackgroundView.backgroundColor = .white
+        addSubview(feedbackBackgroundView)
+
+        feedbackTableView.backgroundColor = .backgroundWhite
+        feedbackTableView.separatorStyle = .none
+        feedbackTableView.showsVerticalScrollIndicator = false
+        feedbackTableView.delegate = self
+        feedbackTableView.dataSource = self
+        feedbackTableView.isScrollEnabled = false
+        feedbackTableView.register(FeedbackMenuTableViewCell.self, forCellReuseIdentifier: FeedbackMenuTableViewCell.reuseIdentifier)
+        feedbackBackgroundView.addSubview(feedbackTableView)
+    }
+
+    private func setupConstraints() {
+        arrowbackgroundView.snp.makeConstraints { make in
+            make.leading.trailing.top.equalToSuperview()
+            make.height.equalTo(size/2)
+        }
+        arrowView.snp.makeConstraints { make in
+            make.trailing.equalTo(arrowbackgroundView.snp.trailing)
+            make.bottom.equalTo(arrowbackgroundView.snp.bottom)
+            make.width.equalTo(size)
+            make.height.equalTo(size)
+        }
+        feedbackBackgroundView.snp.makeConstraints { make in
+            make.top.equalTo(arrowbackgroundView.snp.bottom)
+            make.leading.trailing.bottom.equalToSuperview()
+        }
+        feedbackTableView.snp.makeConstraints { make in
+            make.leading.trailing.top.bottom.equalTo(feedbackBackgroundView)
+        }
+    }
+}
+
+extension FeedbackView: UITableViewDataSource {
+
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        feedbackOptions.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        guard let cell = feedbackTableView.dequeueReusableCell(withIdentifier: FeedbackMenuTableViewCell.reuseIdentifier, for: indexPath) as? FeedbackMenuTableViewCell else { return UITableViewCell() }
+        let feedbackOption = feedbackOptions[indexPath.row]
+        cell.configure(for: feedbackOption)
+        return cell
+    }
+
+}
+
+extension FeedbackView: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        40
+    }
+
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let optionSelected = feedbackOptions[indexPath.row]
+        if (optionSelected == "Send feedback") {
+            let feedbackURL = "https://forms.gle/t9umWjckEs4NNWNS8/"
```

Comment:
Maybe put this link within your Secrets.plist?  Just a suggestion but it feels weird putting a raw URL straight in code.